events/data-anonymization-ethical-ai.md

```diff
@@ -0,0 +1,35 @@
+---
+parent: Events
+title: The Importance of Data Anonymization to Build Ethical AI
+description: Lecture in the Convergence Lecture Series
+location: online
+name: The Importance of Data Anonymization to Build Ethical AI
+startDate: 2023-10-18
+
+seo:
+  type: Event
+  name: The Importance of Data Anonymization to Build Ethical AI
+  startDate: 2023-10-18
+  endDate: 2023-10-18
+  eventAttendanceMode: OnlineEventAttendanceMode
+  eventStatus: EventScheduled
+
+  location:
+    type: VirtualLocation
+    url: https://events.teams.microsoft.com/event/ab293df8-6fd4-4905-a33a-c45e5546ccf4@6b902693-1074-40aa-9e21-d89446a2ebb5
+
+  organizer:
+    type: Organization
+    name: Centre of Translation Studies, University of Surrey
+    url: https://www.surrey.ac.uk/centre-translation-studies
+---
+
+The talk **The Importance of Data Anonymization to Build Ethical AI** will take place online on 18 October, 2023, at 16:00 BST (GMT+1).
+
+The speaker will be Manuel Herranz, CEO of Pangeanic.
```

Comment:
We could hyperlink this to /companies.md#pangeanic

Reply:
done